react-components/config/webpack.config.prod.js

```diff
@@ -260,12 +266,13 @@ module.exports = function(webpackEnv) {
             // `web` extension prefixes have been added for better support
             // for React Native Web.
             extensions: paths.moduleFileExtensions
-                .map(ext => `.${ext}`)
-                .filter(ext => useTypeScript || !ext.includes('ts')),
+                .map((ext) => `.${ext}`)
+                .filter((ext) => useTypeScript || !ext.includes("ts")),
             alias: {
                 // Support React Native Web
                 // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
-                'react-native': 'react-native-web',
+                "react-native": "react-native-web",
+                react: path.resolve("./node_modules/react"),
```

Comment:
For anyone else reviewing, I believe this is the only change in this file.

Reply:
@sri2k1us I have pushed some prettier settings so our IDEs can preserve the styles from the original CRA ejected files.

If you rebase this PR on top of master now and reformat this file with the new settings, it will help clean-up this PR's diff.

Reply:
@psarando - Thanks. That helped with getting the original formatting back but it still changed all double quotes to single :(

Reply:
Updated configs to use double quotes.

Reply:
@sri2k1us: Yes, the configs were intentionally set to use single quotes, because that's how the ejected file was originally formatted by CRA, and it's also how it was formatted before the changes in this PR.

Keeping the original formatting, including the single quotes, will not only clean up this file's diff in this PR, it will also allow us to run an easy diff with the original CRA ejected scripts to help us quickly find our custom changes.

Reply:
🆗 Undid my last commit. Back to single to quotes.